src/components/ItaliaTheme/Blocks/TextCard/CardWithImage/Block.jsx

```diff
@@ -24,7 +24,7 @@ const messages = defineMessages({
   },
 });
 
-const renderImage = (image, showImage, sizeNatural) =>
+const renderImage = (image, showImage, sizeNatural, altText) =>
```

Comment:
```suggestion
const renderImage = (image, showImage, sizeNatural, altText = '') =>
```

Metti qui il default, è più "javascriptoso" e più safe

Reply:
Fatto!